tests/foreman/destructive/test_ansible.py

```diff
@@ -46,3 +46,31 @@ def test_positive_persistent_ansible_cfg_change(target_sat):
     assert param in target_sat.execute(f'cat {ansible_cfg}').stdout.splitlines()
     target_sat.execute('satellite-installer')
     assert param in target_sat.execute(f'cat {ansible_cfg}').stdout.splitlines()
+
+
+def test_positive_import_all_roles(target_sat):
+    """Import all Ansible roles available by default.
+
+    :id: 53fe3857-a08f-493d-93c7-3fed331ed391
+
+    :Steps:
+
+        1. Navigate to the Configure > Roles page.
+        2. Click the `Import from [hostname]` button.
+        3. Get total number of importable roles from pagination.
+        4. Fill the `Select All` checkbox.
+        5. Click the `Submit` button.
+        6. Verify that number of imported roles == number of importable roles from step 3.
+        7. Verify that Ansible variables have been imported along with roles.
+        8. Delete an imported role.
+        9. Verify that the role was successfully deleted.
+
+    :expectedresults: All roles are imported successfully. One role is deleted successfully.
+    """
+    with target_sat.ui_session() as session:
+        assert session.ansibleroles.import_all_roles() == session.ansibleroles.imported_roles_count
```

Comment:
WHY? could you add some description?

Reply:
Updated.

Reply:
@adarshdubey-star can we close this PR as we discussed this last week?

Reply:
@Gauravtalreja1 I saw that case failing once with the last three runs I have executed the build once again if that's green I will close this one.